packages/svelte/src/internal/client/runtime.js

```diff
@@ -1925,6 +1926,7 @@ export function pop(accessors) {
 		current_component_context = context_stack_item.p;
 		context_stack_item.m = true;
 	}
+	return component;
```

Comment:
Related: when you do `bind:this` on a component without accessors you'll get `undefined` which may be confusing because you're thinking "this component didn't instantiate yet". Should we return an empty object in this case? Or do we weigh the theoretical performance overhead of this higher?

Reply:
yeah a default `component = {}` param seems reasonable

Reply:
made that change